CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift

```diff
@@ -581,21 +570,32 @@ public let TYPE_NODES: [Node] = [
     experimentalFeature: .nonescapableTypes,
     nameForDiagnostics: "lifetime specifier",
     documentation: "A specifier that specifies function parameter on whose lifetime a type depends",
+    traits: [
+      "Parenthesized"
+    ],
```

Comment:
The node shouldn’t have the `Parenthesized` trait because the left parenthesis is not the first token. You need to add an XFail for this in the CodeGeneration tests.